scopes/dependencies/dependency-resolver/dependency-resolver.docs.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-description: Simplifies the dependency management of components in a Bit workspace.
-labels: ['core aspect', 'dependencies']
+description: Simplifies the dependency management components in a Bit workspace.
```

Comment:
Scalable and simple dependency resolution for components.

Reply:
done